src/OrchardCore/OrchardCore.Search.Elasticsearch.Core/Services/ElasticsearchIndexManager.cs

```diff
@@ -618,13 +734,24 @@ internal async Task<ElasticsearchResult> SearchAsync(string indexName, string qu
         {
             var fullIndexName = GetFullIndexName(indexName);
 
-            var endpoint = new EndpointPath(Elastic.Transport.HttpMethod.GET, fullIndexName + "/_search");
-            var searchResponse = await _elasticClient.Transport
-                .RequestAsync<SearchResponse<JsonObject>>(endpoint, postData: PostData.String(query));
+            var response = await _elasticClient.Transport
+                .RequestAsync<SearchResponse<JsonObject>>(Elastic.Transport.HttpMethod.GET, $"{fullIndexName}/_search", postData: PostData.String(query), GetDefaultConfiguration());
 
-            if (searchResponse.IsSuccess())
+            if (!response.IsValidResponse)
             {
-                ProcessSuccessfulSearchResponse(elasticTopDocs, searchResponse);
+                if (response.TryGetOriginalException(out var ex))
+                {
+                    _logger.LogWarning("There were issues creating an index in Elasticsearch. Exception: {OriginalException}", ex);
```

Comment:
If there is an error to be logged, use LogError, and the exception is a specific argument then.

Reply:
@sebastienros  I made these changes.